lms/services/lti_hypothesis_bridge.py

```diff
@@ -107,36 +113,29 @@ def wrapper(context, request):  # pylint:disable=too-many-branches
         except HAPIError as err:
             raise HTTPInternalServerError(explanation=err.explanation) from err
         else:
-            upsert_group_info(context, request)
-        return wrapped(context, request)
-
-    return wrapper
-
-
-def add_user_to_group(wrapped):
-    """
-    Add the Hypothesis user to the course group.
-
-    Add the Hypothesis user corresponding to the current request's LTI user, to
-    the Hypothesis group corresponding to the current request's LTI course.
-
-    Assumes that the Hypothesis user and group have already been created.
-
-    Assumes that it's only used on LTI launch views.
-    """
-
-    @functools.wraps(wrapped)
-    def wrapper(context, request):
-        if not context.provisioning_enabled:
-            return wrapped(context, request)
-
-        try:
-            request.find_service(name="h_api_requests").post(
-                f"groups/{context.h_groupid}/members/{context.h_user.userid}"
-            )
-        except HAPIError as err:
-            raise HTTPInternalServerError(explanation=err.explanation) from err
-
-        return wrapped(context, request)
-
-    return wrapper
+            cls._upsert_group_info(context, request)
+
+    @classmethod
+    def _upsert_group_info(cls, context, request):
```

Comment:
This is the function that was originally in `lms/views/decorators/_helpers.py`. It's only called here and is really part of `add_user_to_group()`.